### ISSUE TITLE
finetune the  ViT-SO400M-14-SigLIP-384 CUDA out od mermory

### ISSUE THREAD

**Issue by ctgushiwei** [mlfoundations/open_clip#1065]
When I fine-tune the ViT-SO400M-14-SigLIP-384 model, I encounter an out-of-memory error even with a batch size of 2. My GPU has  24GB of memory.

**Comment by rwightman** [comment]
@ctgushiwei it is what it is, that model requires a lot of memory, reduce the model size and / or sequence length (image size). Gradient checkpointing will help but it will be very slow and probably won't get good results with a batch size that small.
